### PR TITLE
Add poc list feature

### LINKED ISSUE #93: Add list of POCs feature for events

### PR DESCRIPTION
**Pull request by wrjgold**
closes #93

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddEventCommand.java [hunk 1]
```diff
@@ -34,42 +37,48 @@ public class AddEventCommand extends AddCommand {
...
-            + PREFIX_EVENT_CELEBRITY + "Sydney Sweeney ";
+            + PREFIX_EVENT_CELEBRITY + "Sydney Sweeney "
+            + PREFIX_EVENT_CONTACTS + "Alex Yeoh, Bernice Yu";
 
     public static final String MESSAGE_SUCCESS = "New Event added: %1$s";
     public static final String MESSAGE_DUPLICATE_EVENT = "This event  already exists in the address book";
     private final EventName eventName;
     private final Time time;
     private final Venue venue;
     private final String personName;
```
Comment (fusin02) [inline src/main/java/seedu/address/logic/commands/AddEventCommand.java:53]: maybe we can change this to celebrityName to be clearer now that we are adding contacts

**Review by fusin02 — COMMENTED** [review]
Looks good overall, just a minor design suggestion

**Review by fusin02 — APPROVED** [review]

File: src/main/java/seedu/address/logic/commands/AddEventCommand.java (hunk 1 above)
Comment (Haris-Irfan) [reply src/main/java/seedu/address/logic/commands/AddEventCommand.java]: I see what you mean I will make the edits needed.

Comment (wrjgold) [reply src/main/java/seedu/address/logic/commands/AddEventCommand.java]: astute observation, thanks fixed it already

**Comment by Haris-Irfan** [comment]
So all is left is just modifying the delete and edit commands with the POC list right?

**Pull request merged**